src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,17 +1,44 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Remove the eslint-disable comment and ensure the control has an associated label for better accessibility.

Reply:
I understood you, but it is cross, and I think unnecessary use label